website/src/app/kb/administer/upgrading/readme.mdx

```diff
@@ -1 +1,57 @@
+high-availability deployments.
+
+## How versioning works in Firezone
+
+Firezone employees a semantic versioning scheme in the standard
+`MAJOR`.`MINOR`.`PATCH` format with the following structure:
+
+| Version type | Example    | Explanation                                                                                                                                                                                                      |
+| ------------ | ---------- | ---------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| `MAJOR`      | `1`        | Marketing version. Updates to this version generally constitute a new architecture or product. Updated every few years on average.                                                                               |
+| `MINOR`      | `20231001` | API version. Clients and gateways **must** be the same API version to function together. The Firezone admin portal supports the latest **two** API versions at the same time. Updated every 6 months on average. |
+| `PATCH`      | `11`       | Patch version. Represents backwards-compatible bug fixes and updates that are generally recommended for all users. Updated on average every week.                                                                |
+
+All Firezone components follow the structure above, so it is trivial to
+determine which clients work with which gateways and for how long.
+
+## Auto-updates
+
+In general we recommend keeping clients and gateways as up-to-date as possible.
+See below for auto-update details for each client.
+
+<TabsGroup>
+  <TabsItem title="macOS" active>
+    Enable auto-updates in Settings.app
+  </TabsItem>
+  <TabsItem title="iOS">Enable auto-updates in Settings.app</TabsItem>
+  <TabsItem title="Android">Enable app auto-updates in Settings</TabsItem>
+  <TabsItem title="ChromeOS">Enable app auto-updates in Settings</TabsItem>
+  <TabsItem title="Linux">
+    Linux auto-updates are configured by default if you deployed using the Linux
+    client deployment guide.
+  </TabsItem>
+  <TabsItem title="Gateway">
+    Gateway auto-updates are configured by default if you deployed using the
+    Linux client deployment guide.
+  </TabsItem>
+</TabsGroup>
+
+## Upgrading gateways with minimal downtime
+
+Gateways that are deployed in a
+[high availability](/kb/deploy/gateway#high-availability) configuration can be
+rolled over to new versions one-by-one so that clients connected to the gateways
+will automatically reconnect to a healthy replica.
+
+Currently the failover timeout is 120 seconds, so upgrades should be performed
+during a planned maintenance window to ensure end-users experience minimal
+disruption. Other than a possible short-term connection interruption to in-use
+resources, users won't be impacted by upgrading high-availability gateways.
```

Comment:
@firezone/engineering -- is this accurate or should we leave this section out for now?

Reply:
cc @conectado